packages/lit-element/src/lib/updating-element.ts

```diff
@@ -169,7 +169,15 @@ export const defaultConverter: ComplexAttributeConverter = {
         return value === null ? null : Number(value);
       case Object:
       case Array:
-        return JSON.parse(value!);
+        // Do *not* generate exception when invalid JSON is set as elements
+        // don't normally complain on being mis-configured.
+        // TODO(sorvell): Do generate exception in *dev mode*.
+        try {
+          value = JSON.parse(value!);
+        } catch (e) {
+          value = null;
+        }
+        return value;
```

Comment:
You don't really need the return here, though it's good for consistency. I'd consider using assignments and `break`s in the switch instead of early returns.

Reply:
Done.